src/clusterfuzz/_internal/tests/appengine/libs/issue_filer_test.py

```diff
@@ -366,6 +408,36 @@ def test_testcase_metadata_invalid(self):
         ['ClusterFuzz', 'Reproducible', 'Pri-1', 'Stability-Crash', 'Type-Bug'],
         issue_tracker._itm.last_issue.labels)
 
+  def test_testcase_save_exception(self):
+    """Tests issue filing when issue.save exception"""
+    self.mock.get.return_value = CHROMIUM_POLICY_FALLBACK
+    original_save = monorail.issue.Issue.save
+    helpers.patch(self, ['libs.issue_management.monorail.issue.Issue.save'])
+    
+    def my_save(*args, **kwargs):
+        if getattr(my_save, 'count', 1):
```

Comment:
This is a little confusing to read. Why would `'count'` be 1 by default?

Let's rename 'count' to 'raise_exception' and write like so:

```python
if getattr(my_save, 'raise_exception', None):
  raise Exception('Boom!')
else:
  setattr(my_save, 'raise_exception', True)
  return original_save(...)
```

Reply:
We want it raise exception at the first call to save, and the second call to be normal. So I believe something like this works:

```python
        if getattr(my_save, 'raise_exception', True):
            setattr(my_save, 'raise_exception', False)
            raise Exception("Boom!")
        else:
            return original_save(...)
```

Reply:
Ah yep, that's the correct one :)

Reply:
applied :)